2

```diff
@@ -0,0 +1,52 @@
+<!-- <form accept-charset="UTF-8" method="POST" action="/users">
+  <label for="username">Username: </label>
+  <input type="text" id="username" name="user[username]"><br><br>
+  <label for="email">Email: </label>
+  <input type="email" id="email" name="user[email]"><br><br>
+  <label for="password">Password: </label>
+  <input type="password" id="password" name="user[password]"><br><br>
+  <input type="submit" value="Submit"> %>
+  <%# <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token" %> 
+  <%# > %>
+ </form> -->
+
+<!-- <%= form_tag('/users') do %>
+  <%= label_tag 'username'%>
+  <%= text_field_tag 'user[username]' %><br/>
+
+  <%= label_tag 'email' %>
+  <%= email_field_tag 'user[email]' %><br/>
+
+  <%= label_tag 'password'%>
+  <%= password_field_tag 'user[password]'%><br/>
+
+  <%= submit_tag 'Create' %>
+<% end %> -->
+
+<% if @user.errors.any? %>
+  <h3>Correct the following errors!</h3>  
+  <ul>
+    <% @user.errors.full_messages.each do |msg| %>
+      <li><%= msg %></li>
+    <% end %>
+  </ul>  
+<% end %>
+
+<%= form_for @user, class:"user_form" do |f|  %>
```

Comment:
There is no need to specify the `class`. You can put:
`<%= form_for @user do |f|%>`

Reply:
We are using this class for styling using css. Kindly let us know if there is any other way